providers/implementations/rands/seed_src.c

```diff
@@ -109,7 +111,7 @@ static int seed_src_generate(void *vseed, unsigned char *out, size_t outlen,
     entropy_available = ossl_pool_acquire_entropy(pool);
 
     if (entropy_available > 0)
-        memcpy(out, rand_pool_detach(pool), rand_pool_length(pool));
+        memcpy(out, rand_pool_entropy(pool), rand_pool_length(pool));
```

Comment:
Er, this isn't right...  I believe you're looking for `rand_pool_buffer()`

Reply:
You're correct.  It's too late here :(

Reply:
Fixed now.